maluuba-eclipse.importorder

```diff
@@ -0,0 +1,16 @@
+#Organize Import Order
+#Tue Dec 29 11:07:35 EST 2015
+0=\#java
+1=\#javax
+2=\#lombok
+3=\#org
+4=\#org.maluuba
+5=\#com
+6=\#com.maluuba
+7=java
```

Comment:
I made the file according to the [Wiki](http://wiki.maluuba.com/w/index.php?title=Getting_Started_with_NLP#Organize_Imports).  If we want something else, then we should change the Wiki too.